--- ./qpmx/template_static.pro.orig	2018-06-11 20:03:29.125075026 +0200
+++ ./qpmx/template_static.pro	2018-06-11 20:02:40.421112252 +0200
@@ -5,6 +5,7 @@
 else: CONFIG += release
 
 # add modules (from qtbase, qtdeclerative, but only if available)
+QT = 
 QT_WANTS *= core gui widgets network xml sql concurrent dbus qml quick quickwidgets particles
 for(mod, QT_WANTS):qtHaveModule($$mod): QT *= $$mod
 
